static/docs/command-reference/pipeline/show.md

```diff
@@ -35,8 +35,7 @@ instead of stages.
   option is specified) of stage outputs instead of paths to DVC-files.
 
 - `--ascii` - visualize pipeline. It will print a graph (ASCII) instead of a
```

Comment:
put a link to the sub-section below

Reply:
what section? should i add subsections in the example sections?
Now it's hard to link it, because there are no subsections in the example section

Reply:
to the one you created, about the pager ... unless I'm missing something

Reply:
you mean to the options section like this?
https://dvc.org/doc/command-reference/pipeline/show#options

I'm not sure if we can reference it better without a header introduction or i'm missing something :)

Reply:
ok got it

Reply:
no, this section `Paging the output`

Reply:
actually, i just removed the whole message about control, because it doesn't make sense now.. only link to the paging the output section